Formula/w/werf.rb

```diff
@@ -1,17 +1,19 @@
 class Werf < Formula
   desc "Consistent delivery tool for Kubernetes"
   homepage "https://werf.io/"
-  url "https://github.com/werf/werf/archive/refs/tags/v2.10.4.tar.gz"
-  sha256 "5d43462a7022225f819fee2caeca1c1dc851c943da65117e2ec3c122ef7023e8"
+  url "https://github.com/werf/werf/archive/refs/tags/v2.10.5.tar.gz"
+  sha256 "871e7a42b5f5457b8a897178288e2d7a323ae78adee9f2015184238a32e88243"
   license "Apache-2.0"
   head "https://github.com/werf/werf.git", branch: "main"
 
   # This repository has some tagged versions that are higher than the newest
-  # stable release (e.g., `v1.5.2`) and the `GithubLatest` strategy is
-  # currently necessary to identify the correct latest version.
+  # stable release (e.g., `v1.5.2`), so it's necessary to check GitHub releases
+  # instead of the Git tags. However, upstream has set a release with a lower
+  # major version as the "latest" release, so we have to use the
+  # `GithubReleases` strategy until the "latest" release is also the highest.
   livecheck do
     url :stable
-    strategy :github_latest
+    strategy :github_releases
```

Comment:
I think it is just an oversight from upstream, can you file an issue for it?

Reply:
Sure. For the time being, I'll drop the `livecheck` changes from this PR and use the `skip-livecheck` label, so we can get the version bump merged and we can handle any `livecheck` tweaks in a follow-up as needed.

Reply:
I was on the fence about using `GithubReleases` as a temporary workaround in this case, so it makes sense to me 👍

Reply:
Upstream marked 2.10.5 as "latest", so that's taken care of for now. Hopefully they sort out their release process to account for having both v1 and v2 releases going forward.

Reply:
yeah, totally :)